apps/test/unit/code-studio/progressReduxTest.js

```diff
@@ -90,7 +90,9 @@ const stageData = [
     lesson_plan_html_url:
       '//localhost.code.org:3000/curriculum/course3/1/Teacher',
     lesson_plan_pdf_url:
-      '//localhost.code.org:3000/curriculum/course3/1/Teacher.pdf'
+      '//localhost.code.org:3000/curriculum/course3/1/Teacher.pdf',
+    student_lesson_plan_html_url:
+      '//localhost.code.org:3000/curriculum/course3/1/Student'
```

Comment:
this URL looks like it is copying from above what we do for things on curriculum builder. I know its just for a test but it would be good to change this to /s/fake-course/lessons/1/student or something